ocs_ci/helpers/dr_helpers.py

```diff
@@ -2227,12 +2251,22 @@ def get_cluster_set_name():
         list: List of uniq cluster set name
     """
     cluster_set = []
+    current_managed_clusters_list = []
     restore_index = config.cur_index
     config.switch_acm_ctx()
     managed_clusters = ocp.OCP(kind=constants.ACM_MANAGEDCLUSTER).get().get("items", [])
-    current_managed_clusters_list = [
-        cluster_name.ENV_DATA.get("cluster_name") for cluster_name in config.clusters
-    ]
+    for cluster_name in config.clusters:
+        if cluster_name.ENV_DATA.get("rbd_dr_scenario"):
+            current_managed_clusters_list.append(
+                cluster_name.ENV_DATA.get("cluster_name")
+            )
+    dr_cluster_relations = config.MULTICLUSTER.get("dr_cluster_relations", [])
+    if dr_cluster_relations:
+        current_managed_clusters_list = [
+            f"{constants.HYPERSHIFT_ADDON_DISCOVERYPREFIX}-{item}"
```

Comment:
This is applicable for hosted clusters only. Changes are needed when using non hosted clusters for clients.